platform/entanglement/test/services/MoveServiceSpec.js

```diff
@@ -194,6 +196,17 @@ define(
                         .toHaveBeenCalledWith(jasmine.any(Function));
                 });
 
+                it("throws an expection when performed on invalid inputs", function () {
+                    function perform() {
+                        moveService.perform(object, newParent);
+                    }
+
+                    policyService.allow.andReturn(true);
+                    expect(perform).not.toThrow();
+                    policyService.allow.andReturn(false); // Cause validate to fail
```

Comment:
Similar to CopyServiceSpec, spying on the validate method instead of the policy service would tighten the focus of this unit test.  As a bonus, this comment would not being necessary.